media_test_harness/runtests.py

```diff
@@ -6,21 +6,65 @@
 import os
 import sys
 
-from firefox_puppeteer import manifest as puppeteer_manifest
-from firefox_ui_harness import cli
-from firefox_ui_harness.options import FirefoxUIOptions
-from firefox_ui_harness.runners import FirefoxUITestRunner
-from firefox_ui_tests import manifest_all as ui_manifest
-from firefox_ui_tests import resources as ui_resources
+import mozlog
+from marionette import BaseMarionetteTestRunner, BaseMarionetteOptions
+from marionette.marionette_test import MarionetteTestCase
 
 import firefox_media_tests
 from testcase import MediaTestCase
 
-
-class MediaTestOptions(FirefoxUIOptions):
+DEFAULT_PREFS = {
+    'app.update.auto': False,
+    'app.update.enabled': False,
+    'browser.dom.window.dump.enabled': True,
+    # Bug 1145668 - Has to be reverted to about:blank once Marionette
+    # can correctly handle error pages
+    'browser.newtab.url': 'about:newtab',
+    'browser.newtabpage.enabled': False,
+    'browser.reader.detectedFirstArticle': True,
+    'browser.safebrowsing.enabled': False,
+    'browser.safebrowsing.malware.enabled': False,
+    'browser.search.update': False,
+    'browser.sessionstore.resume_from_crash': False,
+    'browser.shell.checkDefaultBrowser': False,
+    'browser.startup.page': 0,
+    'browser.tabs.animate': False,
+    'browser.tabs.warnOnClose': False,
```

Comment:
As long as you are setting default prefs explicitly, why don't you set "browser.tabs.remote.autostart" to True? This is the e10 pref.